msteams-platform/toolkit/TeamsFx-CLI.md

```diff
@@ -1,23 +1,390 @@
+| `teamsapp collaborator grant` | Check user's permission. |
+| `teamsapp collaborator status` | Show permission status for the project. |
+
+### Parameters for `teamsapp collaborator grant`
+
+| Parameter | Required | Description |
+|:----------------  |:-------------|:-------------|
+| `--env` | Yes | Select an existing environment for the project. |
+| `--folder` | No | Select root folder of the project. Defaults to `./`. |
+| `--email` | No | Input email address of collaborator. |
+| `--teams-app-manifest` | No | Manifest of Your Teams app. |
+| `--aad-app-manifest` | No | Manifest of your Microsoft Entra app. |
+
+### Parameters for `teamsapp collaborator status`
+
+| Parameter | Required | Description |
+|:----------------  |:-------------|:-------------|
+| `--env` | Yes | Select an existing environment for the project. |
+| `--folder` | No | Select root folder of the project. Defaults to `./`. |
+| `--teams-app-manifest` | No | Manifest of Your Teams app. |
+| `--aad-app-manifest` | No | Manifest of your Microsoft Entra app. |
+| `--list-all-collaborators` | No | List all collaborators. |
+
+### Scenarios for `teamsapp permission`
+
+* Grant permission
+
+  Project creator and collaborators can use `teamsapp` permission grant command to add a new collaborator to the project:
+
+  ```typescript
+  teamsapp permission grant --env dev --email user-email@user-tenant.com
+  ```
+
+  After you receive the required permission, project creator and collaborators can share the project with the new collaborator by GitHub, and the new collaborator can have all the permissions for Microsoft 365 account.
+
+* Show permission status
+
+  Project creator and collaborators can use `teamsapp collaborator status` command to view Microsoft 365 account permission for specific env:
+
+  ```typescript
+  teamsapp permission status --env dev
+  ```
+
+* List all collaborators
+
+  Project creator and collaborators can use `teamsapp collaborator status` command to view all collaborators for specific env:
+
+  ```typescript
+  teamsapp collaborator status --env dev --list-all-collaborators
+  ```
+
+## `teamsapp update aad-app`
+
+Update the specific application manifest file.
+
+| Command | Description |
+|:----------------  |:-------------|
+| `teamsapp entra-app update` | Update the Microsoft Entra ID app in the current application. |
+| `teamsapp update` | Update the Teams app manifest to Teams Developer Portal. |
+
+### Parameters for `teamsapp entra-app update`
+
+| Parameter | Required |Description |
+|:----------------  |:-------------|:-------------|
+| `--env` | Yes | Select an existing environment for the project. |
+| `--folder` | No | Select root folder of the project. Defaults to `./` |
+| `--entra-app-manifest-file` | No | Enter the Microsoft Entra ID app manifest template file path, it's a relative path to project root folder, defaults to `./aad.manifest.json`. |
+
+### Parameters for `teamsapp update teams-app`
```

Comment:
`teamsapp update teams-app`  is renamed to `teamsapp update`

Reply:
Updated.